_posts/2023-03-21-announcing-ci.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "US-RSE has a new Fiscal Sponsor!"
+tags: [finances]
+author: Julia Damerow
+---
+
+
+We’re excited to announce that US-RSE has switched to a new fiscal sponsor. We are now a fiscally sponsored project of [Community Initiatives](https://communityin.org/) (CI)! Our previous fiscal sponsor, Open Collective Foundation (OCF), represented an important first step allowing us to accept donations and making it possible to reimburse members for expenses they incurred on behalf of US-RSE. 
+Moving forward, Community Initiatives’ additional tools, resources, and infrastructure will better serve US-RSE’s future organizational and financial goals.
+
+[According to Wikipedia](https://en.wikipedia.org/wiki/Fiscal_sponsorship), “[f]iscal sponsorship refers to the practice of non-profit organizations offering their legal and tax-exempt status to groups—typically projects—engaged in activities related to the sponsoring organization's mission.” 
+For US-RSE, that means that by being a project of Community Initiatives, their 501(c)(3) non-profit status extends to us, so that, for instance, we can accept donations without the expense or effort of having to form a non-profit ourselves. 
```

Comment:
```suggestion
For US-RSE, that means that by being a project of CI, their 501(c)(3) non-profit status extends to us, so that, for instance, we can accept donations without the expense or effort of having to form a non-profit ourselves. 
```